404.html

```diff
@@ -16,12 +16,14 @@
 </head>
-
 <body>
+    <div class="cover">
+        <center><img src="/assets/coding.png" width="275dp"></center>
```

Comment:
Use relative link to assets

Reply:
by relative link what do you mean?

Reply:
I've uploaded the image to the assets folder: https://github.com/TurboProgramming/makeroid.github.io/blob/master/assets/coding.png

Reply:
I mean that relative link won't work across other websites like my fork. https://pavi2410.tk/makeroid.github.io

I've replaced most of the links. You just have to remove the "/" from the beginning

Reply:
That image doesn't look good there, IMHO